client/state/selectors/get-site-timezone-string.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Return the site setting `timezone_string` value from the state-tree
```

Comment:
Maybe add a bit more context. It's hard to tell what this is or what it's used for if I looked this up in dev docs search.

How about something like
```
Returns the site's timezone name, in the format of 'America/Araguaina.' See also:
https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
```